plugin-api/src/main/java/org/hyperledger/besu/plugin/data/BlockOverrides.java

```diff
@@ -67,7 +69,8 @@ public BlockOverrides(
       final Optional<Hash> stateRoot,
       final Optional<BigInteger> difficulty,
       final Optional<Bytes> extraData,
-      final Optional<String> mixHashOrPrevRandao) {
+      final Optional<String> mixHashOrPrevRandao,
+      final Optional<String> parentBeaconBlockRoot) {
```

Comment:
does changing this constructor make it a breaking change for the plugin API?

Reply:
We expose a block simulator service that allows plugins to simulate blocks